docs/fields.rst

```diff
@@ -210,10 +209,10 @@ structures and render them appropriately. ::
     '{"billing_address": {"line 1": "123 fake street", "line 2": null, "state": "NY", "zip": "10468", "city": "New York"}, "name": "bob", "shipping_address": {"line 1": "555 nowhere", "line 2": null, "state": "NY", "zip": "10468", "city": "New York"}}'
 
 This example uses two Nested fields.  The ``Nested`` constructor takes a dict of
-fields to render as sub-fields.  The important difference when using ``Nested``
-vs nested dicts as the previous example is the context for attributes.  In
-this example "billing_address" is a complex object that has it's own fields,
-the contexted passed to the nested field is the sub-object instead of the
+fields to render as sub-fields.  The important difference between the ``Nested`` constructor
```

Comment:
Can you make the first reference to `Nested` in this section a reference to the class docs?